src/services/storageProvider/index.js

```diff
@@ -0,0 +1,20 @@
+const service = require('feathers-mongoose');
+const StorageProvider = require('./model');
+const hooks = require('./hooks');
+
+module.exports = function storageProviderService() {
+	const app = this;
+	const options = {
+		Model: StorageProvider,
+		paginate: {
+			default: 1000,
+			max: 1000,
+		},
+		lean: false,
+	};
+
+	app.use('/storageProvider', service(options));
+
+	const storageProviderService = app.service('/storageProvider');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['storageProviderService' is already declared in the upper scope.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)